index.html

```diff
@@ -20,377 +20,272 @@
-            <h1>
-                Hey There.I’m Felix
-            </h1>
+    <section class="base">
+      <div class="nav-bar">
+        <div class="logo">
+          <span>Felix</span>
         </div>
-        <div class="name-2"><h3>I’am a Software Developer</h3></div>
-        <div class="p-name">
-            <p>
-                I can help you build a product, 
-                feature or website Look through some of my work and experience! 
-                If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.
-            </p>
+        <div class="menu">
+          <ul>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li><i class="bi bi-envelope"></i></li>
+          </ul>
+          <i class="bi bi-list"></i>
         </div>
-
-        <section class="social-illustration">
-
-            <div class="socialmedia">
-                <ul>
-                    <li><a href="#"><img src="assets/Vector.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Group.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Vector2.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Vector3.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Vector4.png" alt=""/></a></li>
-                    
-                </ul>
- 
-    
-                
-    
-    
-            </div>
-
-        </section>     
-    </section>
-    <section class="recent-works">
-      <h1>My Recent Works</h1>
-      <div class="line-rule"></div>
-
-      <div class="img-placeholder">
-        <img src="assets/img-placeholder.png" alt=""/>
       </div>
 
-      <div class="post">
-        <h2>Multi-Post Stories</h2>
-      </div>
-      <div class="content">
-        <p>
-          A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
-        </p>
-      </div>
-
-      <div class="tag">
+      <section class="headline">
+        <h1>Hey There.<br>I’m Felix</h1>
+        <span>I’am a Software Developer</span>
+        <p>I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+      </section>
+      <div class="socials">
         <ul>
-          <li>Css</li>
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
+          <li><img src="assets/Vector.png" alt=""/></li>
+          <li><img src="assets/Group.png" alt=""/></li>
+          <li><img src="assets/Vector2.png" alt=""/></li>
+          <li><img src="assets/Vector3.png" alt=""/></li>
+          <li><img src="assets/Vector4.png" alt=""/></li>
```

Comment:
- [ ] [**OPTIONAL**] Good job working on this project however, you should make sure that these links are wrapped in  `a` tag because they represented path to specific accounts. :+1: 

![Screenshot from 2022-12-10 04-10-26](https://user-images.githubusercontent.com/80458199/206824012-ccc8411e-a94f-4c36-b13f-8a6a43b7baf7.png)

![Screenshot from 2022-12-10 04-07-48](https://user-images.githubusercontent.com/80458199/206824021-d85acee6-a765-44e7-a172-71984d6a50da.png)